source/frontend/src/features/mapSideBar/acquisition/tabs/AcquisitionFileTabs.test.tsx

```diff
@@ -78,4 +78,51 @@ describe('AcquisitionFileTabs component', () => {
       expect(getByText('Documents')).toHaveClass('active');
     });
   });
+
+  it('hides the expropiation tab when the Acquisition file type is "Consensual Agreement"', () => {
```

Comment:
please replace all instances of "expropiation" with "expropriation". I see it in comments like this and also on file names, section names, etc. 
Thanks!

Reply:
Updated